packages/shell-api/src/enums.ts

```diff
@@ -26,8 +26,8 @@ export enum DBQueryOption {
 }
 
 export const shellApiType = Symbol.for('@@mongosh.shellApiType');
-// TODO: Would require changes to java-shell
-// export const asPrintable = Symbol('asPrintable');
-export const asShellResult = Symbol.for('@@mongosh.asShellResult');
+// TODO: This requires changes to java-shell.
+export const asPrintable = Symbol.for('@@mongosh.asPrintable');
```

Comment:
I think you can get rid of the TODO here since it's been updated

Reply:
Ah right, thanks for noticing – done! :)